.github/workflows/pre-release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Create Pre-Release
+on:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

Comment:
Can we move this into .github/actions/unit-tests/action.yaml?

Reply:
It was already in the unit tests action but I thought it would be more explicit as a setup in the relevant workflows. Is there a specific preference for keeping it in the unit test action ?
I can put it back without any problem, I just would like to know the rationale behind it :)